services/brig/src/Brig/API/Public/Swagger.hs

```diff
@@ -48,39 +64,27 @@ swaggerPregenUIServer =
 {- FUTUREWORK(fisx): there are a few things that need to be fixed before this schema collection
    is of any practical use!
 
-- `ToSchema` instances of team notifications are wrong.  To do this right, search
-  schema-profunctor tutorial for bind/dispatch, and consult conversation events for
-  examples.
-
-- swagger2 doesn't handle types with the same name from different models well; it silently
-  drops the second definition, which is what you want only if there are no name clashes as
-  in our case with three types called `Event` and three types called `EventType`.  We have
-  solved this by rendering the three event types seperately and returning each
-  declarations list in a super-list.  For a better work-around, check
-  https://github.com/GetShopTV/swagger2/issues/14.
```

Comment:
i'd like to keep this issue open as long as we don't have an automatic check that makes CI red if we create another name clash in the future.  it would be perfect if this check was in the library and could be moved upstream.  for this PR, keeping a FUTUREWORK for this is fine.